_authors/anandabodhi.md

```diff
@@ -1,3 +1,6 @@
 ---
 title: "Ayya Anandabodhi"
 ---
+Ayya Anandabodhi first encountered the Buddha’s teachings in her early teens, igniting a deep interest in the Buddha’s Path of Awakening. She lived and trained as a nun in the Forest Tradition at Amaravati and Chithurst monasteries in England from 1992 until 2009, when she moved to the US to help  establish Aloka Vihara, a training monastery for women, where she now resides.
```

Comment:
I generally prefer the bios to be worded in a short, neutral, factual style. Like an encyclopedia. Not just copy-and-pasted from elsewhere.

For example, "igniting a deep interest" is more evocative language than I would like. Perhaps just remove the first sentence?

Reply:
I understand your point. I will do my best with other authors and try to modify the bios I find, however, I'll appreciate the feedback if you have time since I am not a native English speaker. Thank you for letting me help a little bit.

Reply:
Ah, understood. I will be happy to review your PRs, no worries 😊 Thanks for the help!